js/utils/lib/trumbowyg.js

```diff
@@ -12,26 +14,33 @@ export const editorOptions = {
     'btnGrp-lists',
     ['horizontalRule'],
   ],
+
 };
 
 export function installRichEditor(attachPoint, options = {}) {
-  const opts = {
-    editorOptions: {
-      ...editorOptions,
-      ...options.editorOptions || {},
-    },
-    ...options,
+  const thing = app.localSettings.get('language');
+  const lang = getTrumboLangFileNameByCode(thing);
```

Comment:
Can we use a descriptive variable like "currentLang" instead of "thing"?

Or just do `const lang = getTrumboLangFileNameByCode(app.localSettings.get('language');`

Reply:
Whoops. That was debug code. Thanks for catching. ;D

Reply:
Fixed.